src/test/pages/IsaacApp.test.tsx

```diff
@@ -146,14 +119,9 @@ describe("IsaacApp", () => {
         });
     });
 
-    // TODO implement test data and this test for CS
-    isPhy && it('should show the users number of current assignments in the navigation menu (Physics only)', async () => {
-        renderTestEnvironment();
-        const myAssignmentsBadge = await screen.findByTestId("my-assignments-badge");
-        expect(myAssignmentsBadge.textContent?.includes("4")).toBeTruthy();
-    });
-
-    isCS && it('should show featured news pods before non-featured ones, and order pods correctly based on id (CS only)', async () => {
+    // TODO implement test data & test for "should show the users number of current assignments in the navigation menu"
```

Comment:
Instead of writing this as a comment, you could have a couple of TODO tests, one of them could look like this:

```typescript
it.todo("should show the users number of current assignments in the navigation menu");
```

Reply:
That's a great feature! Changed it now